composer.json

```diff
@@ -22,7 +22,8 @@
     "zendframework/zend-diactoros": "~1.1",
     "container-interop/container-interop": "^1.1",
     "zendframework/zend-math": "~2.2",
-    "zendframework/zend-eventmanager": "~2.2"
+    "zendframework/zend-eventmanager": "~2.2",
+    "zendframework/zend-stdlib": "~2.5"
```

Comment:
Okay I see you're using it for the Options class. I'd really like to remove this useless dependency, maybe you could see if you could not extract the important part of the AbstractOptions class to move it to the options class.